### ISSUE TITLE
Feedback link to GH based on SI users logged in

### ISSUE THREAD

**Issue by roman87** [TeamMentor/TM_4_0_Design#829]
In talking to @craSH  he had a great idea to replace the "send us an email" feedback link with link to open a Github issue, iff the user is logged in with SI email address. This would work for both content and code issues. 

The link could also contain a page reference or some other information that would go into the body of the issue, making it easier to write. And GH allows this by doing something like
```
https://github.com/TeamMentor/Master/issues/new?title=Title&body=%3CURL%20of%20parent%20page%3E
```

@michaelhidalgo wanna take a shot at this for 4.1

**Comment by DinisCruz** [comment]
Humm, this means that we will need to detect if the user's email belong to @securityinnovation.com , but at the moment the TM_Graph only captures the username.

This would easily be done when we have the UserData locally (https://github.com/TeamMentor/TM_4_0_Design/issues/180) but in the short term it could be done by capturing the user's data on login (since at that moment in time we have a valid 3.5 token)

**Comment by roman87** [comment]
Well, we could return the email address from 3.5 on successful login.

**Comment by romichg** [comment]
Moving this up as this is the next importaint thing

**Comment by michaelhidalgo** [comment]
Github url should be https://github.com/TeamMentor/TM_4_0_Design/issues, this is a configurable value so it can be changed eventually.

**Comment by romichg** [comment]
One more thing, if the issue is for the article ( link at the bottom of the page in full article view) then it should go to https://github.com/TMContent/Lib_UNO/issues/<correct URL to open a new issue> (Also configurable)
The title of the issue should contain the article title. The body of the issue should contain the article guid.

**Comment by romichg** [comment]
This looks great on Jade. Needs to be added to Flare.

**Comment by michaelhidalgo** [comment]
Currently waiting on https://github.com/TeamMentor/TM_4_0_Design/issues/961

**Comment by romichg** [comment]
#961 is done, lets get this added.

**Comment by michaelhidalgo** [comment]
With Pull Request.

 Submodules : 

`4cc3a0c002fad180f86d32ccab22fc74998c675e build/TM_4_0_JSON_Data (4cc3a0c)
 4321dcc3823fd3abc0e5b910a1d8118ac61a215c build/TM_Gulp (heads/master)
+e537dea90b12de1217947008fed3e6b36e53f335 code/TM_Angular (heads/Issue_829_Feeback)
+f6aa69bcfde0ba0214bee22e6acb8865492940e1 code/TM_Flare (4.0.3-335-gf6aa69b)
 120056e6043656b86698ad46983041dd5aa89782 code/TM_GraphDB (4.0.1-36-g120056e)
 8bf0b3c7da2a7c5e654a8ead211e36d124b4d974 code/TM_Jade (remotes/origin/Dev)
 c8406320efbbdb8986823fce6935499c0f7ae470 code/TM_Shared (remotes/origin/Dev)
 7ab139c0b438a4cae7312346cfdce2b2c823e55e code/TM_Static (remotes/origin/Dev)
+f93f83ed5c30f258e9718d614d47fc844d6010af code/TM_Website (4.0.3-144-gf93f83e)
 7f6f1cad93bd9e6668ef9e0d0a06939df225bf76 config/SiteData_TM (remotes/origin/Dev)
 dd65a11bf320940d515c8e06870616a5ae8a03f1 data/Lib_Docs-json (heads/master)
 256c437c3cc25c33025918bf12d93ae42c011e22 data/Lib_UNO (remotes/origin/Beta)
 95e9f6fc9d672de1d36b4bf3d5026a12158d362a data/Lib_UNO-json (remotes/origin/Beta)
 921ea5e009deca34bc3b34fbd7e54ec746f0fa87 qa/TM_QA (4.0.1-34-g921ea5e)`

**Comment by romichg** [comment]
This is fixed
